backend/src/main/java/ai/verta/modeldb/utils/CommonHibernateUtil.java

```diff
@@ -257,6 +258,7 @@ public void runLiquibaseMigration(DatabaseConfig config)
     runLiquibaseMigration(
         config,
         liquibaseRootFilePath,
-        new FileSystemResourceAccessor(new File(System.getProperty(CommonConstants.USER_DIR))));
+        new FileSystemResourceAccessor(new File(System.getProperty(CommonConstants.USER_DIR))),
+        Optional.of("liquibase/reset_filepath_database_change_log_2022_10.json"));
```

Comment:
nit: maybe put a day or something more precise?

Reply:
no strong opinion on this one. I think the idea (@ad-47 can answer for sure) was that this was a patch added for the 2022_10 release, but as I said, I'm good with adding more granularity if people think it would be good.

Reply:
I have added a name like that because this fix/bug is catching up in 2022_10 months. so we can track the timeline using the name I have added it. I am also open to changing it with more concrete if we have something in mind.

Reply:
we can leave